docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[Kubernetes-dashboard]]
+= The Kubernetes dashboard: Linux session data from your Kubernetes clusters
+
+This page explains how to deploy a https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/[daemonset] to collect session data from your Kubernetes clusters using the {endpoint-cloud-sec} integration, and how to view that data in Kibana’s Kubernetes dashboard.
+
+**Prerequisites**: This feature requires Elastic Stack version 8.4 or higher. Your Elastic deployment must have the {endpoint-cloud-sec} integration {security-guide}/install-endpoint.html[enabled], and you need an active {fleet-guide}/fleet-overview.html[{fleet} Server].
+
+**Support Matrix**: This feature is currently available on GKE and EKS using Linux hosts and Kubernetes versions that match the following specifications:
+|=====================
+| | **Kubernetes versions** | **Container runtime** | **Nose OSes**
+|**EKS**| 1.18; 1.19; 1.20; 1.21| Docker | Amazon Linux 2, Bottlerocket OS
+|**GKE**| Regular (default channel): 1.21 and 1.22; Stable: 1.20 and 1.21; Rapid: 1.22 and 1.23 | containerd | Container-optimized OS (COS), Ubuntu
+|=====================
+You first need to download the daemonset manifest .yaml, then modify it to include your {fleet} URL and Enrollment Token before you deploy it to the clusters you want to monitor. After that, data will start to appear in the Kubernetes dashboard.
+
+[discrete]
+== Download and modify the daemonset manifest
+The daemonset integrates Elastic Endpoint into your Kubernetes cluster. The Elastic Agent is enrolled to a running Fleet Server using the `FLEET_URL` parameter, and connected to a specific Elastic Agent policy using the `FLEET_ENROLLMENT_TOKEN`.
+
+1. Download the daemonset manifest http://github.com/elastic/placeholder[here] for GKE or http://github.com/elastic/placeholder[here] for EKS.
```

Comment:
Apologies for not mentioning this before. I see a slight hiccup here.
The YAML file will container placeholders for the user to fill in with regards to:
1) Agent container image
2) Endpoint container image
3) K8sMD container image 

They should be getting these images from the public repository:
docker.elastic.co/endpoint/k8smd:<version>
docker.elastic.co/endpoint/endpoint-security:<version>
docker.elastic.co/beats/elastic-agent:<version>

Reply:
Thanks!